minesweeper.py

```diff
@@ -63,13 +71,13 @@ def do_step(self, arg_string):
             return False
 
         if game_end:
-            print()
-            print(self.board)
             print()
             if game_end == EndState.VICTORY:
                 print('You win!!')
             elif game_end == EndState.DEFEAT:
                 print('You lose.')
+            with self.term.location(0, 1):
+                print(self.board)
```

Comment:
It's silly that this duplicates the board print below, and sillier that it prints with different coordinates.

Reply:
If we can't control the scroll behavior, is Blessed really the right display package at all?